app/controllers/concerns/scholars_archive/works_controller_behavior.rb

```diff
@@ -74,6 +76,35 @@ def destroy_all_files
 
     private
 
+    # METHOD: Manually add controlled_vocab object to funding body
+    def store_funding
```

Comment:
I like this, but maybe we meet up and think up a neat solution about how we could do this differently.